domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -65,6 +65,7 @@
     private final ClaimDocumentCollection claimDocumentCollection;
     private final LocalDate claimantResponseDeadline;
     private final ClaimState state;
+    private final ClaimSubmissionOperationIndicators claimSubmissionOperationIndicators;
```

Comment:
You may need to add this in `@JsonIgnoreProperties` to  ignore them when deserialize

Reply:
Also we would require to have an Optional getter for this, as existing claims won't have this object.

Reply:
Have added the @jsonignoreproperties